src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -51,6 +51,7 @@
 #include "base/utils/string.h"
 #include "gui/autoexpandabledialog.h"
 #include "gui/uithememanager.h"
+#include "gui/torrentcategorydialog.h"
```

Comment:
Please move it one line up to be sorted alphabetically.

Reply:
Moved it up!